{
    "CVE_data_meta": {
        "ASSIGNER": "cybersecurity@schneider-electric.com",
        "ID": "CVE-2019-6809",
        "STATE": "PUBLIC",
        "UPDATED": "2020-06-15"
    },
    "cna-container": {
        "affected": {
            "vendors": [
                {
                    "products": [
                        {
                            "versions": [
                                {
                                    "version_value": "firmware version prior to V2.90"
                                }
                            ]
                        },
                        {
                            "versions": [
                                {
                                    "version_value": "firmware version prior to V3.10"
                                }
                            ]
                        },
                        {
                            "versions": [
                                {
                                    "version_value": "all versions"
                                }
                            ]
                        },
                        {
                            "versions": [
                                {
                                    "version_value": "all versions"
                                }
                            ]
                        }
                    ],
                    "vendor_name": "Schneider Electric SE"
                }
            ]
        },
        "descriptions": [
            {
                "lang": "eng",
                "value": "A CWE-248: Uncaught Exception vulnerability exists in Modicon M580 (firmware versions prior to V2.90), Modicon M340 (firmware versions prior to V3.10), Modicon Premium (all versions), Modicon Quantum (all versions), which could cause a possible denial of service when reading invalid data from the controller."
            }
        ],
        "problemtypes": {
            "descriptions": [
                [
                    {
                        "CWE-ID": "CWE-248: Uncaught Exception",
                        "description": "CWE-248: Uncaught Exception",
                        "lang": "eng",
                        "type": "CWE"
                    }
                ]
            ]
        },
        "references": [
            {
                "name": "https://www.schneider-electric.com/en/download/document/SEVD-2019-134-11/",
                "refsource": "CONFIRM",
                "url": "https://www.schneider-electric.com/en/download/document/SEVD-2019-134-11/"
            }
        ]
    },
    "data_format": "MITRE",
    "data_type": "CVE",
    "data_version": "5.0"
}